src/main/java/frc/robot/RobotMap.java

```diff
@@ -59,6 +59,7 @@ public static class mapClimber {
   // MOTORS: 50 -> 59
   public static class mapTransfer {
     public static final int TRANSFER_MOTOR_CAN = 50;
+    // public static final int DigitalInput = 51;
```

Comment:
I would uncomment this out and reference it when you create the Digital input in Transfer